src/content/docs/ai-observability/advanced-customization/drop-sensitive-data.mdx

```diff
@@ -0,0 +1,306 @@
+1. Create a filter name. Because one kind of data requires four drop rules, we recommend a naming convention that helps you track the events you're dropping data from. 
+1. Paste in the base NRQL query:
+  ```sql
+  SELECT <COLUMN> FROM <EVENT_TYPE> WHERE <COLUMN> RLIKE <REGEX>
+  ```
+1. Referring to the table, update the `<COLUMN>` and `<EVENT_TYPE>` placeholders to match the attributes and events. For example:
+  ```sql
+  SELECT content FROM LlmChatCompletionMessage WHERE content RLIKE <REGEX>
+  ```
+1. Add the regex that corresponds to the kind of information you want to drop. For example, if you're targeting IP addresses, the completed query should read:
+  ```sql
+  SELECT content FROM LlmChatCompletionMessage WHERE content RLIKE ^([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})$
+  ```
+1. Repeat the above steps to create drop filters for the remaining events and column pairs. 
+
+## Your data compliance responsibilities [#responsibilities]
+
+New Relic can't guarantee that this functionality completely resolves your data disclosure concerns, nor can we provide support for building out your NRQL queries. We recommend that you:
+
+* Review your drop rules and confirm they're accurately identifying and discarding the data you want dropped.
+* Check that your drop rules are still dropping sensitive information after you've created them. 
+
+While drop rules ensure that personal information about your end users aren't stored in NRDB, creating the rules themselves imply the kinds of data you maintain, including the format of your data or systems. This is important when determining control permissions for certain users in your org, as certain permissions let users view and edit all information in the rules you create.
+
+## How drop filters work [#drop-rules-work]
+
+A drop filter directs the agent to drop information if it meets a set of conditions. A drop filter contains three parts:
+
+* Events: A stored record from an interaction within your system.
+* Attributes, or columns: A key-value pair attached to data objects. 
+* Regex: A string of characters and operators that corresponds to kinds of information. 
+
+### Events and attributes 
+
+In a typical AI interaction, a prompt or request undergoes certain processes (like embedding) that are recorded as discrete events. For example, let's say a customer requests a street address on file. The prompt is processed by the model, which pulls additional context through various services and databases. Your AI assistant then returns with a response that contains the requested information. 
+
+Below are the different events that occur in a system in an interaction like this: 
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Event
+            </th>
+            <th>
+                Attribute, or column 
+            </th>
+            <th>
+                Description 
+            </th>
+        </tr>
+    </thead>
+    <tbody>
```

Comment:
There would be an `LlmChatCompletionSummary`, 2 or more `LlmChatCompletionMessage` (at least the user question and the model response), then the `LlmEmbedding`. Potentially there will be an `LlmTool` (depends on if they are using LangChain). It is unlikely to include a `LlmFeedbackMessage` immediately unless the customer has a feedback process and has wired the New Relic agent up to it; at least nothing in the prior paragraph mentions that there should be a feedback message.

I get what you are going for here, but it gets messy quickly :)

Reply:
I do want to point out that while the `LlmChatCompletionSummary` even is generated, it is also not in the list of events for the drop rule as that event does not contain any user or LLM generated content.

Reply:
I reframed this to place the responsibility of knowing what their data looks like on the user, with a note that different environments will require different kinds of drop filters (per this message). I also added a note that LlmChatCompletionMessage may require at least two drop filters per your comment as well. Hopefully this is adequate enough